docs/dapps/getting-started.md

```diff
@@ -45,19 +53,149 @@ Next, change the workspace to the tutorial project:
 cd alephium-tutorial
 ```
 
-and compile your contracts:
+Have a look in the `contracts/` folder, you can find `token.ral`:
+
+```rust
```

Comment:
I thought we could do https://www.stevemar.net/github-code-in-readme/ but it doesn't seems to work yet, probably because we aren't in https://github.com/github/linguist yet